lib/coverband/configuration.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'rubygems'
```

Comment:
Don't think this is needed.

Reply:
I think it is required for old versions of Ruby, like 1.9 maybe.  If we aren't worried about going back that far then it can be removed.

Reply:
Right. Coverband won’t work that far back anyway.

Reply:
Done